src/services/Animation.service.ts

```diff
@@ -13,6 +14,7 @@ const MIXAMO_SCALE = 0.01;
 
 const fbxLoader = new FBXLoader();
 const gltfLoader = new GLTFLoader();
+gltfLoader.setMeshoptDecoder(MeshoptDecoder);
 
```

Comment:
New mesh opt decoder lets us allow to use compressed and quantized meshes in visage saving up to 50% of geometry size in glbs.